For Ember Times No.107: Add Ember Auto Import Section + Contributors (#192)
ember-learn/ember-blog: source/2019-07-19-the-ember-times-issue-107.md

## [Section Title in Title Case 🐹](#section-url)
<change section title emoji>
<consider adding some bold to your paragraph>

<add your name to author list, top and bottom>
<add blurb and emoji to "SOME-INTRO-HERE">

---

> ## [Custom Configuration of Ember Auto Import for Your Project 🐹](https://github.com/ef4/ember-auto-import/tree/v1.3.0#customizing-build-behavior)

Reviewer: Did you want to use the custom emoji above?

Author: ```suggestion
## [Custom Configuration of Ember Auto Import for Your Project ⏬](https://github.com/ef4/ember-auto-import/tree/v1.3.0#customizing-build-behavior)
```

> The useful addon [Ember Auto Import](https://emberobserver.com/addons/ember-auto-import) helps you to use third-party JavaScript libraries in your Ember app easily by providing a **zero-config import** mechanism for **npm** modules.

Reviewer: I think you can reorder the words to make your persuasion carry a more active voice.
```suggestion
[Ember Auto Import](https://emberobserver.com/addons/ember-auto-import), a very useful addon, helps you easily use third-party JavaScript libraries in your Ember app by providing a **zero-config import** mechanism for **npm** modules.
```

> You can [configure the addon to fit your project's specific needs](https://github.com/ef4/ember-auto-import#customizing-build-behavior). By providing the `autoImport` configuration object in your app options defined in `ember-cli-build.js`, you can, f.e., change the name of the packages to import from or exclude packages to be imported with Ember Auto Import, which you might have already provided to your app in another way.

Reviewer 2: I feel a bit of a contradiction between the `zero-config` and this sentence.

Author: Good point! I shortened the previous sentence to de-emphasise the zero config bit, so that the configuration section makes more sense just as you suggested :sparkles:

> You can [configure the addon to fit your project's specific needs](https://github.com/ef4/ember-auto-import#customizing-build-behavior). By providing the `autoImport` configuration object in your app options defined in `ember-cli-build.js`, you can, f.e., change the name of the packages to import from or exclude packages to be imported with Ember Auto Import, which you might have already provided to your app in another way.

Reviewer: I don't think f.e. is a common way to abbreviate "for example." I'd spell out the phrase for readers whose native tongue is not English.
```suggestion
You can [configure the addon to fit your project's specific needs](https://github.com/ef4/ember-auto-import#customizing-build-behavior). By providing the `autoImport` configuration object in your app options defined in `ember-cli-build.js`, you can, for example, change the name of the packages to import from or exclude packages to be imported with Ember Auto Import, which you might have already provided to your app in another way.
```

> If you're already an avid user of Ember Auto Import (or plan to become one), be aware that from version 1.4.0+ the addon will **no longer include** Webpack's polyfills for a set of Node.js globals and modules automatically. [When upgrading to a version of the addon beyond 1.4.0](https://github.com/ef4/ember-auto-import#i-upgraded-my-ember-auto-import-version-and-now-things-dont-import-what-changed) you might see errors such as `global is not defined` popping up in your app or test suite. This might indicate that some of the packages included in your app may - even though they're run in a browser environment - still rely on globals that are usually only available in a Node environment and that these might require a polyfill to work correctly. See also [this issue on the Ember Auto Import repo](https://github.com/ef4/ember-auto-import/issues/218) for more context.

Reviewer: I really liked the description that you gave to help readers understand why they might see the `global is not defined` error. I would like to see the description kept, but divided into two paragraphs for ease of reading.
```suggestion
If you're already an avid user of Ember Auto Import (or plan to become one), be aware that from version 1.4.0+ the addon will **no longer include** Webpack's polyfills for a set of Node.js globals and modules automatically.

[When upgrading to a version of the addon beyond 1.4.0](https://github.com/ef4/ember-auto-import#i-upgraded-my-ember-auto-import-version-and-now-things-dont-import-what-changed), you might see errors such as `global is not defined` in your app or test suite. This might indicate that some of your packages—even though they run in a browser environment—still rely on globals (these are usually only available in a Node environment) and may require a polyfill to work correctly. You can check [this issue on the Ember Auto Import repo](https://github.com/ef4/ember-auto-import/issues/218) for more context.
```
